lwt-unix/jbuild

```diff
@@ -0,0 +1,19 @@
+(library
+ ((name        conduit_lwt_unix)
+  (public_name conduit-lwt-unix)
+  (preprocess (pps (ppx_sexp_conv)))
+  (wrapped false)
+  (modules (resolver_lwt_unix conduit_lwt_unix conduit_lwt_server conduit_lwt_tls conduit_lwt_unix_ssl conduit_lwt_launchd))
+  (libraries (conduit-lwt tls uri.services ipaddr.unix logs 
+    (select conduit_lwt_launchd.ml from
+     (launchd  -> conduit_lwt_launchd_real.ml)
+     (!launchd -> conduit_lwt_launchd_dummy.ml))
+    (select conduit_lwt_unix_ssl.ml from
+     (lwt.ssl  -> conduit_lwt_unix_ssl_real.ml)
```

Comment:
hm, leaving this alone for the moment, since the depopts have to remain `ssl` for compatibility.  In a future revision we can move the whole lot to use `lwt-ssl` consistently, as there will be no further depopts left except for the sole use of `ssl`